static/sass/styles.scss

```diff
@@ -163,3 +163,12 @@
 .u-row-selected {
   background-color: $color-mid-x-light;
 }
+
+.u-instance-status {
+  font-size: 80%;
+  font-style: italic;
+}
+
+.u-button-icon {
```

Comment:
This shouldn't be needed.

The misplacement comes from the button being inside the h4, which shouldn't be the case. The h4 should only wrap the instance name.

Maybe don't use the `BaseLayout` here?

Reply:
Done @ [4f62c9e](https://github.com/canonical/lxd-ui/commit/4f62c9e2df86d2b692c4298deac8212f25ffcc02)